### ISSUE TITLE
Options with getCredentialDefinitions function

### ISSUE THREAD

**Issue by anthonyherveHCS** [IBM-Blockchain-Identity/openssi-websdk#23]
Hello,
I try to use the CredentialDefinitionQueryParams in getCredentialDefinitions API (https://github.com/IBM-Blockchain-Identity/openssi-websdk/blob/5d5746ca6b5b8d7937ebfaae65b0847f960cad9d/nodejs/sdk.js#L579).
`let cred_def = await agent.getCredentialDefinitions(false, {schema_name: 'My_Schema_Name'});`
But it always returns an empty array.
While when i use the following curl command, it works 
`curl -u 'agentId:agentPWD' -X GET 'https://agency.ibmsecurity.verify-creds.com/api/v1/credential_definitions?schema_name=My_Schema_Name' `

Thank you.

**Comment by joevolleyball** [comment]
Started looking at this today.  The switch between our first tech preview and our second has brought a structure change.  `schema_name` is no longer a property on a credential definition.  If you change this to `schema.name`, it should work for you.  This is what I've added to the openssi-websdk tests that works for me:

```
// validate that filtering will also find it
definitions = await secondIssuer.getCredentialDefinitions(false, {"schema.name": locationSchema.name});
expect(definitions.find(d => d.id === locationCredentialDefinition.id)).to.not.be.undefined;
```

I've updated the comments in the sdk.js to use `schema.name`.  I also updated comments to use `$neq` as that seems to be the proper way to filter on "not equals" now.  More information here: https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0050-wallets/README.md#wallet-query-language
